spyder/widgets/mixins.py

```diff
@@ -694,13 +694,10 @@ def get_line_separator(self):
     def get_text_with_eol(self):
         """Same as 'toPlainText', replace '\n'
         by correct end-of-line characters"""
-        utext = to_text_string(self.toPlainText())
-        lines = utext.splitlines()
+        utext = self.toPlainText()

```

Comment:
```suggestion
        text = self.toPlainText()
```

Now that we're not transforming the text that comes from `toPlainText` to unicode, there's no need to use the `u` prefix.